core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -323,6 +325,11 @@ var AdminPage = function() {
     }
   };
 
+  this.createDummyClassroom = async function() {
+    await action.click('Activities Tab', activitiesTab);
```

Comment:
Are you sure the activities tab actually exists on prod mode? The Activities tab in the admin page is hidden in that mode, you might need to create the necessary constructs manually (see other tests).

Reply:
(I think this might be the cause of the failure here: https://github.com/oppia/oppia/actions/runs/3500183810/jobs/5862586972 )

Reply:
Updated